plugins/template-validator.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const validator = require('screwdriver-template-validator');
+
+exports.register = (server, options, next) => {
+    server.route({
+        method: 'POST',
+        path: '/validator/template',
+        config: {
+            handler: (request, reply) => {
+                // rip eslint
+                const result = validator(request.payload.yaml)
+                .then((stuff) => {
+                    reply(stuff);
+                }, (validationErr) => {
+                    reply({
+                        error: validationErr
```

Comment:
The `screwdriver.yaml` needed a default case because otherwise the job won't even start. For this, why do we need a default config?